StopIteration:  (virtualenv/run/plugin/discovery.py:22) with 0.13.0

```
$ pyb -vX
[DEBUG] Registered Python environment 'system': CPython version 3.9.7.final.0 on linux in ['/home/username/.pyenv/versions/3.9.7/envs/project/bin/python3.9']
PyBuilder version 0.13.0.dev20210912232320
Build started at 2021-09-15 01:31:58
------------------------------------------------------------
[DEBUG] Loading project module from ./build.py
[DEBUG] Setting up plugins VEnv at '/home/username/Documents/src/karellen/project/.pybuilder/plugins/cpython-3.9.7.final.0'
Traceback (most recent call last):
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/cli.py", line 461, in main
    reactor.prepare_build(property_overrides=options.property_overrides,
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/reactor.py", line 203, in prepare_build
    self._setup_plugin_directory(reset_plugins, no_venvs)
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/reactor.py", line 512, in _setup_plugin_directory
    plugin_env = per["pybuilder"] = PythonEnv(plugin_dir, self).create_venv(with_pip=True,
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/python_env.py", line 214, in create_venv
    create_venv(self._env_dir,
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/python_env.py", line 419, in create_venv
    virtualenv.cli_run(args, setup_logging=False)
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/_vendor/virtualenv/run/__init__.py", line 30, in cli_run
    of_session = session_via_cli(args, options, setup_logging, env)
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/_vendor/virtualenv/run/__init__.py", line 48, in session_via_cli
    parser, elements = build_parser(args, options, setup_logging, env)
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/_vendor/virtualenv/run/__init__.py", line 69, in build_parser
    discover = get_discover(parser, args)
  File "/home/username/.pyenv/versions/3.9.7/envs/project/lib/python3.9/site-packages/pybuilder/_vendor/virtualenv/run/plugin/discovery.py", line 22, in get_discover
    default=next(iter(choices)),
StopIteration
------------------------------------------------------------
BUILD FAILED - StopIteration:  (virtualenv/run/plugin/discovery.py:22)
------------------------------------------------------------
Build finished at 2021-09-15 01:31:58
Build took 0 seconds (59 ms)
```

--- a/build.py
+++ b/build.py
@@ -136,8 +136,8 @@ def initialize(project):
     project.set_property("filter_resources_target", "$dir_dist")
     project.get_property("filter_resources_glob").append("pybuilder/__init__.py")
     project.include_file("pybuilder", "LICENSE")
-    project.include_file("pybuilder._vendor", "LICENSES")
-    project.include_file("", "*.whl")  # All included binary wheels from vendors
+    project.include_directory("pybuilder/_vendor", "*",
+                              "$dir_source_main_python")  # All included vendored payload is included
 
     project.set_property("sphinx_doc_author", "PyBuilder Team")
     project.set_property("sphinx_doc_builder", "html")

